src/base/datetime/calendar/header/index.ts

```diff
@@ -0,0 +1,295 @@
+import { html, property, svg, query } from "lit-element";
+import {
+  KucBase,
+  dispatchCustomEvent,
+  CustomEventDetail
+} from "../../../kuc-base";
+import { en, zh, ja } from "../../resource/locale";
+
+type BaseDateTimeCalendarHeaderProps = {
```

Comment:
@dong0 
type BaseDateTimeCalendarHeaderProps using for constructor.
For internal component, we don't need to provide constructor function and we can remove it.